src/SetupModel.h

```diff
@@ -1,15 +1,17 @@
 #ifndef COVIDSIM_SETUPMODEL_H_INCLUDED_
 #define COVIDSIM_SETUPMODEL_H_INCLUDED_
 
-void SetupModel(char*, char*, char*, char*);
-void SetupPopulation(char*, char*);
+#include <string>
+
+void SetupModel(std::string const&, std::string const&, std::string const&, std::string const&, std::string const&, std::string const&, std::string const&);
```

Comment:
Comments on function parameters please.  Especially those that take multiple strings, and also indicate which ones are allowed to be empty, and what that means (that we don't load/save/do a calculation).

Reply:
I've documented `SetupModel.h` to the best of my understanding of what each function does. Feel free to point out inaccuracies/inconsistencies.